domain/infosync/resource_manager_client.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2022 PingCAP, Inc.
```

Comment:
```suggestion
// Copyright 2023 PingCAP, Inc.
```

Reply:
It's a rename of an old file with some changes. Github regards it as a totally new file.